src/models/visits.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model } = require('mongoose');
+
+const visitSchema = Schema({
+	clientId: {
+		type: Schema.Types.ObjectId,
+		ref: 'Client',
+	},
+	date: {
+		type: Date,
+	},
+	type: {
+		type: String,
+	},
+	description: {
+		type: String,
+	},
+	price: {
+		type: Number,
+	},
+	status: {
+		type: String,
+	},

```

Comment:
The same about ```{ versionKey: false, timestamps: true }```.

Reply:
I think it`s no need adding object with configuration to `Visits` model.

Reply:
I think we don't need versionsKey, but some moment we can need createdAt and updatedAt.